checker/src/main/java/org/checkerframework/checker/calledmethods/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,50 @@
+package org.checkerframework.checker.calledmethods.framework;
+
+import com.sun.source.tree.NewClassTree;
+import javax.lang.model.element.ExecutableElement;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+/** add CalledMethods annotations to code generated by framework */
+public interface FrameworkSupport {
+
+    /**
+     * Allows for supporting a framework's {@code toBuilder} routine. Typically, the returned
```

Comment:
This doesn't say what the method is or when it is called.  A discussion of intended use is helpful, but first describe the method.